target/linux/ramips/dts/mt7628an_qding_qc202.dts

```diff
@@ -0,0 +1,116 @@
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "qding,qc202", "mediatek,mt7628an-soc";
+	model = "Qding QC202";
+
+	leds {
+		compatible = "gpio-leds";
+
+		gpio-relay {
+			label = "gpio-relay";
+			gpios = <&gpio 42 GPIO_ACTIVE_HIGH>;
+		};
+
+		ethernet {
+			label = "ethernet";
+			gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi {
+			label = "wifi";
+			gpios = <&gpio 44 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps {
+			label = "reset";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;

```

Comment:
A reset button should have a linux,code of KEY_RESTART for the reboot and firstboot functionality.